modules/tinymce/src/plugins/help/main/ts/ui/KeyboardNavTab.ts

```diff
@@ -3,68 +3,90 @@ import { Dialog } from 'tinymce/core/api/ui/Ui';
 // TODO: When we translate this, we should pull the HTML out into a HTML file, the way TBIO does it.
 // That requires webpack and rollup changes though, so inlining it for now.
 /* eslint-disable max-len */
-const description = `<h1>Editor UI keyboard navigation</h1>
+const description = `<h1>Begin keyboard navigation</h1>
 
-<h2>Activating keyboard navigation</h2>
+<dl>
+  <dt>Focus the Menu bar</dt>
+  <dd>Windows or Linux: Alt+F9</dd>
+  <dd>macOS: &#x2325;+F9</dd>
+  <dt>Focus the Toolbar</dt>
+  <dd>Windows or Linux: Alt+F10</dd>
+  <dd>macOS: &#x2325;+F10</dd>
+  <dt>Focus the footer</dt>
+  <dd>Windows or Linux: Alt+F11</dd>
+  <dd>macOS: &#x2325;+F11</dd>
```

Comment:
Apologies for the last minute suggestion but the `+` shouldn't be there on macOS. Neither Apple, nor our own menus, include it.
```suggestion
  <dd>Windows or Linux: Alt+F9</dd>
  <dd>macOS: &#x2325;F9</dd>
  <dt>Focus the Toolbar</dt>
  <dd>Windows or Linux: Alt+F10</dd>
  <dd>macOS: &#x2325;F10</dd>
  <dt>Focus the footer</dt>
  <dd>Windows or Linux: Alt+F11</dd>
  <dd>macOS: &#x2325;F11</dd>
```